test/functional/transactions/0.tx.js

```diff
@@ -0,0 +1,637 @@
+				done();
+			});
+			badTransactions.push(transaction);
+		});
+
+		it('using float fee should fail', function (done) {
+			transaction.fee = 1.2;
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.match(/Invalid transaction body - Failed to validate transaction schema: Expected type integer but found type number/);
+				done();
+			});
+			badTransactions.push(transaction);
+		});
+
+		it('using positive overflown fee should fail', function (done) {
+			transaction.fee = 1298231812939123812939123912939123912931823912931823912903182309123912830123981283012931283910231203;
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.be.not.ok;
+				node.expect(res).to.have.property('message').to.have.string('Invalid transaction body - Failed to validate transaction schema: Value 1.2982318129391238e+99 is greater than maximum 10000000000000000');
+				done();
+			});
+			badTransactions.push(transaction);
+		});
+
+		it('using null recipientId should fail', function (done) {
+			transaction.recipientId = null;
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				// TODO
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction id');
+				done();
+			}, true);
+			badTransactions.push(transaction);
+		});
+
+		it('using undefined recipientId should fail', function (done) {
+			transaction.recipientId = undefined;
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				// TODO
+				node.expect(res).to.have.property('message').to.equal('Invalid transaction id');
+				done();
+			}, true);
+			badTransactions.push(transaction);
+		});
+
+		it('using NOT string recipientId should fail', function (done) {
+			transaction.recipientId = 1;
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.match(/^Invalid transaction body - Failed to validate transaction schema: Expected type string but found type /);
+				done();
+			}, true);
+			badTransactions.push(transaction);
+		});
+
+		it('using NOT allowed string recipientId should fail', function (done) {
+			transaction.recipientId = '15738697512051092602'; //Address without L invalid
+
+			sendTransaction(transaction, function (err, res) {
+				node.expect(res).to.have.property('success').to.not.be.ok;
+				node.expect(res).to.have.property('message').to.match(/^Invalid transaction body - Failed to validate transaction schema: Object didn't pass validation for format address: /);
+				done();
+			}, true);
+			badTransactions.push(transaction);
+		});
+
+		it('using string recipientId smaller than expected recipientId should fail', function (done) {
```

Comment:
so shorter not smaller ;) but address never can have 1 char - we should adjust that check in core